airflow/www/views.py

```diff
@@ -543,7 +543,7 @@ class AirflowBaseView(BaseView):
     }
 
     if not conf.getboolean('core', 'unit_test_mode'):
-        extra_args['sqlite_warning'] = settings.Session.bind.dialect.name == 'sqlite'
+        extra_args['sqlite_warning'] = settings.Session.bind.dialect.name == 'sqlite'  # type: ignore
```

Comment:
This will not be needed if we manage to resolve https://github.com/apache/airflow/pull/20096#pullrequestreview-825466253. (Not saying you should remove it right now, just a reminder to track changes there and adapt this PR is that’s merged first.)

Reply:
Here, it cant resolve that bind is part of Session

Reply:
```suggestion
        extra_args['sqlite_warning'] = settings.engine.dialect.name == 'sqlite'
```

Reply:
Which I have just merged, so you can rebase now @khalidmammadov